rebar.config

```diff
@@ -70,6 +70,11 @@
             {include_erts, true}
         ]}
     ]},
+    {test, [
+        {deps, [
+            {proper, ".*", {git, "https://github.com/manopapad/proper.git", master}}
```

Comment:
how come we ended up adding a dependency? is it because this library is used only for testing?